extend/flytekit-xgboost/flytekitplugins/xgboost/task.py

```diff
@@ -25,9 +26,9 @@ class ModelParameters(object):
         verbose_eval: If verbose_eval is True then the evaluation metric on the validation set is printed at each boosting stage.
     """
 
-    num_boost_round: int = 10
+    num_boost_round: int = 4
```

Comment:
xgboost, by default, uses '10' for `num_boost_round`. We'll have to modify this number in our example.